indico/migrations/versions/20230515_1145_cb46beecbb93_add_editable_is_deleted.py

```diff
@@ -0,0 +1,32 @@
+"""Add Editable.is_deleted
+
+Revision ID: cb46beecbb93
+Revises: 5d05eda06776
+Create Date: 2023-05-15 11:45:47.447151
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'cb46beecbb93'
+down_revision = '5d05eda06776'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('editables', sa.Column('is_deleted', sa.Boolean(), nullable=False, server_default='false'),
+                  schema='event_editing')
+    op.alter_column('editables', 'is_deleted', server_default=None, schema='event_editing')
+    op.drop_constraint('uq_editables_contribution_id_type', 'editables', schema='event_editing', type_='unique')
```

Comment:
Do we need the `type_='unique'`? Docs say it's only required for MySQL

Reply:
thanks! stuff leftover from `--autogenerate` 😅